redesign-project/src/styles.scss

```diff
@@ -0,0 +1,446 @@
+body {
+    margin: 0;
+}
+
+/* For mobile: */
```

Comment:
In an ideal mobile-first design, most of the mobile styles would be the base styles and then overridden by media queries for larger screens.
I definitely wanted to see some Sass partials used here and think it would help keep your styles more structured. 🙌